roles/openshift_master_facts/tasks/main.yml

```diff
@@ -73,3 +73,9 @@
       oauth_templates: "{{ openshift_master_oauth_templates | default(None) }}"
       oauth_always_show_provider_selection: "{{ openshift_master_oauth_always_show_provider_selection | default(None) }}"
       image_policy_config: "{{ openshift_master_image_policy_config | default(None) }}"
+      builddefaults_http_proxy: "{{ openshift_builddefaults_http_proxy | default(None) }}"
+      builddefaults_https_proxy: "{{ openshift_builddefaults_https_proxy | default(None) }}"
+      builddefaults_no_proxy: "{{ openshift_builddefaults_no_proxy | default(None) }}"
+      builddefaults_git_http_proxy: "{{ openshift_builddefaults_git_http_proxy | default(None) }}"
+      builddefaults_git_https_proxy: "{{ openshift_builddefaults_git_https_proxy | default(None) }}"
+      builddefaults_config: "{{ openshift_builddefaults_json | default(builddefaults_yaml) }}"
```

Comment:
I didn't get this sorted out today. Will work on this tomorrow when i can sync with either of you on irc.